### PR TITLE
Add two missing bot settings to default.json

### PR DESCRIPTION
**Pull request by jasonblais**
`DisableBotsWhenOwnerIsDeactivated` was added to model/config.go by https://github.com/mattermost/mattermost-server/commit/06b579d18aa3bb5d1aa960f4a3261c9d3d430cb4#diff-2279f86634da3f787f279961af76713a

`CreateBotAccounts` was added to model/config.go by https://github.com/mattermost/mattermost-server/commit/30061df0360489f5ebf0be4236be8ec2182f0c3e#diff-2279f86634da3f787f279961af76713a

If this needs more work than just adding to default.json, please let me know.

### CODE REVIEW

File: config/default.json [hunk 1]
```diff
@@ -77,7 +77,9 @@
         "DisableLegacyMFA": true,
         "EnableEmailInvitations": false,
         "ExperimentalLdapGroupSync": false,
-        "ExperimentalStrictCSRFEnforcement": false
+        "ExperimentalStrictCSRFEnforcement": false,
+        "CreateBotAccounts": false,
+        "DisableBotsWhenOwnerIsDeactivated": true
```
Comment (jasonblais) [inline config/default.json]: I noticed `EnableBotAccountCreation` is actually `CreateBotAccounts`, which is inconsistent with the model/config.go naming convention. Should this be changed?

Comment (crspeller) [reply config/default.json]: Yes. Ticket please. Needs to be 5.12.

Comment (jasonblais) [reply config/default.json]: Ticket: https://mattermost.atlassian.net/browse/MM-15662

**Comment by jasonblais** [comment]
Thanks Jesse, the defaults do indeed match: https://github.com/mattermost/mattermost-server/blob/master/model/config.go#L636

```go
	if s.DisableBotsWhenOwnerIsDeactivated == nil {
		s.DisableBotsWhenOwnerIsDeactivated = NewBool(true)
	}

	if s.CreateBotAccounts == nil {
		s.CreateBotAccounts = NewBool(false)
	}
```

@crspeller if the changes look okay, I would like to merge as-is and worry about renaming the config setting separately via https://mattermost.atlassian.net/browse/MM-15662

**Comment by amyblais** [comment]
@crspeller Was this cherry-picked to v5.12?

**Pull request merged**